### DISCUSSION TITLE
Facing Problem...

### DISCUSSION THREAD

**Discussion by JdumB** [aliencaocao/netease_cloudmusic_discord_rpc#28] [Q&A] [answered]
Hi there I can't get what I want... Do I miss anything? Thanksss
 What I did: 
Download Netease.Cloudmusic.Discord.RPC.exe  // facing error (a referral was returned from the server) 
Right click .exe -> Properties -> Compalibility -> Run this as an administrator -> Apply // error above gone after this setup
Right click .exe -> Run as administrator

Here is the debug.log
![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/09877fb3-0b0b-4e8d-9d9f-0c6968ed0dde)
Thanks

**Comment by JdumB** [comment]
![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/b6f9954a-b8a5-4073-bc80-7465f48527fc)

**Comment by aliencaocao** [comment]
Yea 2.10.12 isnt supported yet, i will try to support it soon

**Reply by JdumB** [reply]
I see. I will downgrade it first. Thanks for your effort! Salute

**Reply by JdumB** [reply]
![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/eaea6d7c-2378-466e-ae63-f3a2caa6f67a)
Hi there, sry to disturb again... but it still no working.

**Reply by aliencaocao** [reply]
It shows it has found the process, so it should work. What are you seeing? Or are you playing any song? Try playing a song and see if it appers in the log

**Reply by JdumB** [reply]
![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/7e22b3b6-c36a-4ffc-8eaa-3012c58027f2)
I change a song here
![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/a8a6fd01-aada-4acd-9ec4-ef77d2d43006)
but nothing change in the log

![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/d3462ac5-0085-4e22-a7d2-44a3ba9f1f57)
I did all here but I checked the previous Q&A mention about 当前隐私状态. I didnt do anything on that... will that affect it? Don't sure what the privacy status talking about.

**Reply by JdumB** [reply]
![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/f03f418d-c628-45c6-9da9-2c4bb58dff20)

**Reply by aliencaocao** [reply, marked as answer]
If you turned on invisible mode (the one showing you as offline even when you are online), then yes it wont show in discord.
But this should not affect the RPC program. It should still show in the log.
Can you try V0.2.8 at https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/releases/download/0.2.8/Netease.Cloudmusic.Discord.RPC.exe ? Just to rule out any issue with the new V0.3.0 since I changed some things to support the new beta client (which still dont work yet but shouldn't affect the non beta V2.x clients that you are using)

**Reply by JdumB** [reply]
Thanks pal ! It works now, but FYI the delog is empty. Thanks a lot~
![image](https://github.com/aliencaocao/netease_cloudmusic_discord_rpc/assets/148857992/7794be41-0bf2-40f2-8ad9-eafe7e4b6bd6)

**Reply by aliencaocao** [reply]
yea the V0.2.8 version has a known bug with debug log which is fixed in V0.3.0.
I wonder what gone wrong with V0.3.0 though, I will test with the latest 2.10.12 netease client and see
